.gitlab-ci.yml

```diff
@@ -54,6 +54,9 @@ generate-rtl:
 
 vsim-compile:
   stage: build
+  extends:
+    - .cache-deps
+    - .init-env
```

Comment:
The `vsim-compile` job gets the generated RTL sources (including the `.bender` folder, where some of them are) as an artifact, so technically this is not even needed I think

Reply:
I added this as the CI was failing, but I don't think it is needed, will revert.